src/AssemblerMarkerGraph.cpp

```diff
@@ -224,6 +225,18 @@ void Assembler::createMarkerGraphVertices(
                 csv << coverage << "," << frequency << "\n";
             }
         }
+
+        if (minCoverage == 0) {
+            try {
+                shasta::PeakFinder p;
+                p.findPeaks(histogram);
+                minCoverage = p.findXCutoff(histogram);
+                cout << "MarkerGraph.minCoverage inferred during run time: " << minCoverage << '\n';
+            }
+            catch (PeakFinderException){
+                throw runtime_error("ERROR: No significant cutoff found in distribution");
```

Comment:
When I make the changes in the assembler options description I will modify the message to be more explanatory. You can leave it as is for now.